docs/source/getting_started/changelog.md

```diff
@@ -0,0 +1,110 @@
+# Changelog
```

Comment:
Is adding this changelog.md an intended change?

Reply:
Happens as part of the built process of the docs. While working on it, I tested it locally, hence the updated file. In principle, it can be removed.